sdk/tests/conftest.py

```diff
@@ -444,7 +444,10 @@ def generic_router_version(
         id=2,
         created_at=datetime.now() + timedelta(seconds=20),
         updated_at=datetime.now() + timedelta(seconds=20),
-        router=None,
+        router=turing.generated.models.Router(
+            environment_name="test_env",
+            name="test_router"
+        ),
```

Comment:
This is done to accommodate changes to how the `RouterVersion` SDK class stores a copy of the `environment_name` and (router) `name`. It keeps these information so that users can easily extract a `RouterConfig` SDK (which as of present contains these fields) that can be used to 1) create new routers, 2) update routers or 3) create new router versions. 

In the future when (2) gets refactored and no longer requires the `environment_name` and (router) `name` anymore, these fields will only serve the creation of new routers (1).